### ISSUE TITLE
`fileUrl` doesn't return a torrent or magnet link

### ISSUE THREAD

**Issue by sucrecacao** [rmartin16/qbittorrent-api#78]
Hi all, 

From the [qbittorrent API](https://github.com/qbittorrent/qBittorrent/wiki/WebUI-API-(qBittorrent-4.1)#get-search-results) , the `fileUrl` attribute from the search result should return a torrent or magnet link.

After a few tries, the values I got from `fileUrl` was always the same as the `descrLink` attribute. 

Is this normal behavior?

**Comment by rmartin16** [comment]
Hello @sucrecacao,
I have not used the search functionality much myself; however, the qBittorrent team appears to have [deferred the actual searching](https://github.com/qbittorrent/qBittorrent/blob/master/src/base/search/searchhandler.cpp) to teams with more familiarity with each search engine. Searching specific search engines is implemented via python scripts and those scripts return the values that are returned from `qbt_client.search.results`. So, I think if the URLs are not what you're expecting, the most likely issue is with the python script returning the wrong value. The management of these scripts, though, appear quite unorganized.....so, Im not too sure where to look for these scripts on github or elsewhere.....hope this help, though...

**Comment by sucrecacao** [comment]
Yes, depending on the search plugin ( i.e. torrent source website ) I get a magnet link or the URL of the torrent webpage. 

In case of the former, I'll just beautiful soup and try to find the magnet link in the webpage URL. 

Would have been more clean to get a magnet link every time from the API, but my hacky solution should not be hard to implement. I just need to write a parser for every plugin I use.
